src/ray/stats/stats.h

```diff
@@ -109,6 +109,7 @@ static inline void Shutdown() {
   }
   metrics_io_service_pool->Stop();
   opencensus::stats::StatsExporter::Shutdown();
+  opencensus::stats::DeltaProducer::Get()->Shutdown();
```

Comment:
Should `DeltaProducer` shutdown before StatsExporter::Shutdown?

Reply:
so that we can flush and export right? I think that's a good idea.

Reply:
> so that we can flush and export right? I think that's a good idea.

It can flush metric at last emit moment.